ChiModules/LinearBoltzmannSolver/SweepChunks/lbs_sweepchunk_pwl.cc

```diff
@@ -0,0 +1,316 @@
+#include "lbs_sweepchunk_pwl.h"
+
+#include "chi_log.h"
+extern ChiLog& chi_log;
+
+#include "chi_mpi.h"
+extern ChiMPI& chi_mpi;
+
+#include "ChiMath/chi_math.h"
+extern ChiMath& chi_math_handler;
+
+//###################################################################
+/**Constructor.*/
+LinearBoltzmann::SweepChunkPWL::
+  SweepChunkPWL(std::shared_ptr<chi_mesh::MeshContinuum> grid_ptr,
+                SpatialDiscretization_PWLD& discretization,
+                std::vector<LinearBoltzmann::CellLBSView>& cell_transport_views,
+                std::vector<double>& destination_phi,
+                const std::vector<double>& source_moments,
+                LBSGroupset& in_groupset,
+                const TCrossSections& in_xsections,
+                const int in_num_moms,
+                const int in_max_num_cell_dofs)
+                    : SweepChunk(destination_phi, false),
+                      grid_view(std::move(grid_ptr)),
+                      grid_fe_view(discretization),
+                      grid_transport_view(cell_transport_views),
+                      q_moments(source_moments),
+                      groupset(in_groupset),
+                      xsections(in_xsections),
+                      num_moms(in_num_moms),
+                      num_grps(in_groupset.groups.size()),
+                      max_num_cell_dofs(in_max_num_cell_dofs),
+                      a_and_b_initialized(false),
+                      spls_index(0),
```

Comment:
It looks like spls_index and angle_set_index are member variables that are essentially loop counters.  Is that intentional?

Reply:
Mmmmm. Interesting. This is not my style of coding (or was it). Let me just confirm with Dominic.

My opinion here is that if they are loop counters then defining them as members lures a new developer into thinking that they have special usage other than just being loop counters. In other words, I would prefer them being in the loop declaration.

Maybe they were used to make the loop declaration fit within 80 characters in which case I would prefer renaming/typedefing

Reply:
@dominiccaron feel free to weigh in here

Reply:
This is not my doing. I made (silently) the same observation a few times in the past when working with the class or deriving from it. I have the same preferences as those mentioned here.

Reply:
Cool. It was probably me. And glad that there is concurrence. I will fix it.

Reply:
Fixed